### PR TITLE
Remove Presence

### PR DESCRIPTION
**Pull request by MateusOFCZ**
The presence was renewed for the new Oglle website, so the logo, thumbnail, description, tags, links, among others, were changed!

### CODE REVIEW

File: Oglle/dist/metadata.json [hunk 1]
```diff
@@ -1,26 +1,18 @@
...
+	"author": {
+		"name": "Leprechaun",
+		"id": "394147438006370304"
+	},
+	"service": "Oglle",
+	"description": {
+        "pt_BR": "Oglle é uma rede social, onde qualquer um pode utiliza-la, aqui você pode fazer amigos e publicações, venha participar é grátis!",
+        "en": "Oglle is a social network, where anyone can use it, here you can make friends and publications, come and participate is free!"
+	},
+	"url": ["oglle.com.br"],
```
Comment (Bas950) [inline Oglle/dist/metadata.json:11]: URL shouldnt be in an array if its only 1

File: Oglle/dist/metadata.json [hunk 2]
```diff
@@ -1,26 +1,18 @@
...
+	"version": "1.3",
+	"logo": "https://i.imgur.com/MqPQK6Y.png",
```
Comment (Bas950) [inline Oglle/dist/metadata.json:13]: version 1.4.0

File: Oglle/dist/presence.js [hunk 3]
```diff
@@ -1,50 +1,60 @@
 var presence = new Presence({
-  clientId: "641614088215986186"
+    clientId: "641614088215986186",
+    mediaKeys: false
```
Comment (Bas950) [inline Oglle/dist/presence.js:3]: it was good like it was before, mediakeys are removed, so should no longer be added.

**Comment by doomlerd** [comment]
Closing in 4 days.

**Comment by MateusOFCZ** [comment]
Hello, you can remove the presence of the PreMiD Store and close the PR!

File: Oglle/dist/metadata.json [hunk 4]
```diff
@@ -1,26 +1,18 @@
...
+		"id": "394147438006370304"
+	},
+	"service": "Oglle",
+	"description": {
+        "pt_BR": "Oglle é uma rede social, onde qualquer um pode utiliza-la, aqui você pode fazer amigos e publicações, venha participar é grátis!",
+        "en": "Oglle is a social network, where anyone can use it, here you can make friends and publications, come and participate is free!"
+	},
+	"url": "oglle.com.br",
+	"version": "1.3.0",
+	"logo": "https://i.imgur.com/MqPQK6Y.png",
```
Comment (Bas950) [inline Oglle/dist/metadata.json:13]: 1.3.0 is no increase from 1.3.1

File: Oglle/dist/metadata.json [hunk 5]
```diff
@@ -1,26 +1,18 @@
...
+	"thumbnail": "https://i.imgur.com/7m0Xfhr.jpg",
+	"color": "#787878",
+	"tags": ["random", "oglle", "apps", "programas", "rede social", "amigos", "amizade", "comunidade"] ,
+	"category": "other"
```
Comment (Bas950) [inline Oglle/dist/metadata.json:17]: you are not allowed to have a space in a tag

File: Oglle/dist/presence.js [hunk 6]
```diff
@@ -1,50 +1,59 @@
...
+    }
+    if(window.location.pathname.includes("/admin")) {
+        presenceData.details = "Painel De Administração"
+    }
+    if(window.location.pathname.includes("/wp-login")) {
+        presenceData.details = "Login Painel"
+    }
+    if(window.location.pathname.includes("/wp-admin")) {
+        presenceData.details = "Painel De Administração"
+    }
```
Comment (Bas950) [inline Oglle/dist/presence.js:56]: ever heard of else if?

**Comment by MateusOFCZ** [comment]
### **Hello, you can remove the presence of the PreMiD Store and close the PR!**

**Comment by Bas950** [comment]
> ### **Hello, you can remove the presence of the PreMiD Store and close the PR!**

you can aswell, just remove the files xD
but why tho, and not just fix these simple errors?

**Comment by MateusOFCZ** [comment]
**The problem is having to convert everything from JS to TS, I don't have time for that, I'll leave the presence like that then, thank you.**

**Pull request merged**